### ISSUE TITLE
Hide very long lists of collaborators in the people selector

### ISSUE THREAD

**Issue by brianherbert** [ushahidi/crowdmap-issues#173]
For users following hundreds of people, settings pages and map creation is a bit insane since all users are displayed. Find a way to reduce the visual clutter.

**Comment by evansims** [comment]
This is also resulting in very long page load times as we have to query each of those individual users, hit CrowdmapID for avatar info, etc. Need to streamline this whole process.

**Comment by brianherbert** [comment]
Yeah, I was going to say bonus points for speeding that up server side.
